content/en/monitors/status/status_page.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Monitor Status Page
+description: "Get an overview of your monitor status over time"
+code_lang: status_page
+type: multi-code-lang
+weight: 1
+aliases:
+- /monitors/monitor_status/
+- /monitors/manage/status
+further_reading:
+- link: "/monitors/status/graphs/"
+  tag: "Documentation"
+  text: "Learn more about monitor graphs"
+- link: "/monitors/status/events/"
+  tag: "Documentation"
+  text: "Learn more about event details"
+---
+
+<div class="alert alert-warning">The provisional status page has limited support for monitors and their features. For more details, see <a href="#restrictions-of-provisional-status-page">Restrictions of provisional status page</a>.<br><br>If you are using the legacy status page, see the <a href="/monitors/status/status_legacy">Status Page (Legacy)</a> documentation</div>
+
+## Overview
+
+When a monitor alerts, the first place to start your investigation is the Monitor Status page. This page displays the graphs and related events to show you why your monitor is in an alert status, so you can understand what is happening around a monitor alert. Further, you can access quick actions to help you move the investigation towards resolution. Use the monitor status page to:
+
+- Review all the context you need to start an investigation
+- Investigate the possible cause of an alert
+- Take action to escalate, maintain, or resolve your investigation
+
+## Header
+
+{{< img src="/monitors/status/status_page_header.png" alt="Monitor status page header" style="width:100%;" >}}
+
+The header contains the monitor's metadata including:- Alert status
+- Monitor status
+- Monitor type
+- Creation date
+- Author
+- Associated teams (if available)
+- Tags
+- Monitor quality actions
```

Comment:
- Monitor quality actions
We might hide this for the moment (sorry for the confusion)

Reply:
```suggestion
```